lib/ts/recipe/dashboard/api/userroles/addRoleToUser.ts

```diff
@@ -0,0 +1,52 @@
+import { APIInterface, APIOptions } from "../../types";
+import UserRoles from "../../../userroles";
+
+import STError from "../../../../error";
+
+const addRoleToUser = async (
+    _: APIInterface,
+    tenantId: string,
+    options: APIOptions,
+    __: any
+): Promise<{
+    status: "OK" | "UNKNOWN_ROLE_ERROR" | "ROLE_ALREADY_ASSIGNED";
+}> => {
+    const requestBody = await options.req.getJSONBody();
+
+    const userId = requestBody.userId;
+    const role = requestBody.role;
+
+    if (role === undefined || typeof role !== "string") {
+        throw new STError({
+            message: "Required parameter 'role' is missing or has an invalid type",
+            type: STError.BAD_INPUT_ERROR,
+        });
+    }
+
+    if (userId === undefined || typeof userId !== "string") {
+        throw new STError({
+            message: "Required parameter 'userId' is missing or has an invalid type",
+            type: STError.BAD_INPUT_ERROR,
+        });
+    }
+
+    const response = await UserRoles.addRoleToUser(tenantId, userId, role);
```

Comment:
what if the roles recipe is not initialised? We need to return `FEATURE_NOT_ENABLED_ERROR` too..